analyses/10_mofa/11_easier.R

```diff
@@ -65,38 +67,34 @@ getFeat <- function(dataobj,
       )
     }
 
+    # Tumor purity ESTIMATE
+    estimate <- immunedeconv::deconvolute(dataobj$tpm[[dataset]], method = "estimate")
+
     # Quantification of pathway activity scores with RPOGENy
-    pathway <- compute_pathways_scores(
-      RNA.counts = dataobj$count[[dataset]],
-      remove.genes.ICB_proxies = remove.genes.ICB_proxies
+    pathway <- compute_pathway_activity(
+      RNA_counts = dataobj$count[[dataset]],
+      remove_sig_genes_immune_response = remove.genes.ICB_proxies
     )
 
     # Quantification of TF activity scores with DoRothEA
     TF <- compute_TF_activity(
-      RNA.tpm = dataobj$tpm[[dataset]],
-      remove.genes.ICB_proxies = remove.genes.ICB_proxies
+      RNA_tpm = dataobj$tpm[[dataset]],
+      regulon_net = "dorothea"
```

Comment:
Just for me to understand, this is basically the only line that is changed between the collectri and dorothea versions?

Reply:
Yes!